rclcpp/test/rclcpp/test_timers_manager.cpp

```diff
@@ -296,12 +238,12 @@ TEST_F(TestTimersManager, timers_thread)
 
   // Run timers thread for a while
   timers_manager->start();
-  std::this_thread::sleep_for(5ms);
+  std::this_thread::sleep_for(50ms);
   timers_manager->stop();
 
   EXPECT_LT(1u, t1_runs);
   EXPECT_LT(1u, t2_runs);
-  EXPECT_EQ(t1_runs, t2_runs);
+  EXPECT_LE(std::abs(t1_runs - t2_runs), 1u);
```

Comment:
technically `1u` needs to be `1` since this changes `*_runs` into integer?

Reply:
Correct.  Fixed in 37edae0.